docs/source/en/model_doc/bark.mdx

```diff
@@ -0,0 +1,96 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Bark
+
+## Overview
+
+The `Bark` model is a transformer-based text-to-audio model proposed by Suno's AI team in [suno-ai/bark](https://github.com/suno-ai/bark). 
+
+
+`Bark` is made of 4 main modules:
+
+- A `semantic model ` (also named `text model`), i.e a causal autoregressive transformer (GPT2-like), which takes into input a tokenized text
```

Comment:
Here we should reference the model names in the modelling file, and explain how they tie into the more descriptive names, e.g.
```markdown
[`BarkSemanticModel`] (also referred to as the 'text' model): a causal auto-regressive transformer model that takes as input tokenized text (and optionally a speaker prompt), and predicts semantic text tokens that capture the meaning of the text 
```